.travis.yml

```diff
@@ -14,9 +14,9 @@ env:
   global:
     - TEST_BLACKLIST=moveit,moveit_ros,moveit_runtime,moveit_ros_perception  # mesh_filter_test fails due to broken Mesa OpenGL
   matrix:
-    - ROS_DISTRO=kinetic ROS_REPO=ros              TEST=clang-format
-    - ROS_DISTRO=kinetic ROS_REPO=ros
-    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed
+    - ROS_DISTRO=melodic ROS_REPO=ros              TEST=clang-format
+    - ROS_DISTRO=melodic ROS_REPO=ros
```

Comment:
I'm not sure why we are testing agianst both the `ci` and `shadow-fixed-ci` debians but I am fairly confident that we can remove this failing test.

Reply:
> I'm not sure why we are testing agianst both the `ci` and `shadow-fixed-ci` debians

it's a form of (almost reverse) regression testing: new changes in other packages (staged in the shadow repository) will be tested against the current state of your package(s).

Without that test you'll only know something is broken (either in your package(s) or in others) *after* everything has been synced to the main repositories.

Probably not what you want.

Reply:
Another possible fix would be to change the rosparam_shortcuts build_depend to  `<build_depend version_gte="0.3.1">rosparam_shortcuts</build_depend>`

Reply:
I'm not sure how that would fix things. If the package version that this depends on is only available in the staging repository, nothing will change (`version` specs in manifests are not checked by CMake/Catkin).

Reply:
I guess that would be a regression. 

I was hoping that we could use the rosdep install call in moveit_ci to update rosparam_shortcuts to 0.3.1 but that does not seem to work on my local machine.